libs/langchain-community/src/vectorstores/neo4j_vector.ts

```diff
@@ -4,10 +4,20 @@ import type { EmbeddingsInterface } from "@langchain/core/embeddings";
 import { VectorStore } from "@langchain/core/vectorstores";
 import { Document } from "@langchain/core/documents";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Any = any;
```

Comment:
Ha, you can just do `/* eslint-disable @typescript-eslint/no-explicit-any */` at the top if you prefer